web/src/beta/lib/core/engines/Cesium/Feature/Tileset/hooks.ts

```diff
@@ -516,11 +530,53 @@ export const useHooks = ({
       : null;
   }, [isVisible, tileset, url, type, meta]);
 
+  const imageBasedLighting = useMemo(() => {
+    if (
+      !property?.specularEnvironmentMaps &&
+      !property?.sphericalHarmonicCoefficients &&
+      !sceneProperty?.light?.specularEnvironmentMaps &&
+      !sceneProperty?.light?.sphericalHarmonicCoefficients
+    )
+      return;
+
+    const ibl = new ImageBasedLighting();
+    const specularEnvironmentMaps = property?.specularEnvironmentMaps
+      ? property?.specularEnvironmentMaps
+      : sceneProperty?.light?.specularEnvironmentMaps;
+    const sphericalHarmonicCoefficients = property?.sphericalHarmonicCoefficients
+      ? property?.sphericalHarmonicCoefficients
+      : sceneProperty?.light?.sphericalHarmonicCoefficients;
+    const imageBasedLightIntensity =
+      property?.imageBasedLightIntensity ?? sceneProperty?.light?.imageBasedLightIntensity;
```

Comment:
```suggestion
    const specularEnvironmentMaps =
      property?.specularEnvironmentMaps ?? sceneProperty?.light?.specularEnvironmentMaps;
    const sphericalHarmonicCoefficients =
      property?.sphericalHarmonicCoefficients ?? sceneProperty?.light?.sphericalHarmonicCoefficients;
    const imageBasedLightIntensity =
      property?.imageBasedLightIntensity ?? sceneProperty?.light?.imageBasedLightIntensity;
```
Maybe we can align the expression of the code?

Reply:
Updated. 🙏🏻